### PR TITLE
Update Staker's documents

### PR DESCRIPTION
**Pull request by notJoon**
# Description

- Update docs for staker

### CODE REVIEW

File: staker/calculate_pool_position_reward.gno [hunk 1]
```diff
@@ -57,6 +57,19 @@ var (
 	lastCalculatedBalance uint64
 )
 
+// CalcPoolPosition calculates and updates the position of pools and rewards for stakers.
```
Comment (r3v4s) [inline staker/calculate_pool_position_reward.gno:60]:
```suggestion
// XXX: need to improve nested iteration
// CalcPoolPosition calculates and updates the position of pools and rewards for stakers.
```

File: staker/external_token_list.gno [hunk 2]
```diff
@@ -6,13 +6,24 @@ import (
 	"gno.land/r/gnoswap/consts"
 )
 
+// defaultAllowed is the list of default allowed tokens
```
Comment (r3v4s) [inline staker/external_token_list.gno:9]:
```suggestion
// defaultAllowed is the list of default allowed tokens to create external incentive
```

File: staker/mint_stake.gno [hunk 3]
```diff
@@ -42,7 +57,7 @@ func MintAndStake(
 
 		// send it over to position to wrap
 		banker := std.GetBanker(std.BankerTypeRealmSend)
-		banker.SendCoins(consts.STAKER_ADDR, consts.POSITION_ADDR, std.Coins{{"ugnot", int64(ugnotSent)}})
+		banker.SendCoins(consts.STAKER_ADDR, consts.POSITION_ADDR, std.Coins{{Denom: "ugnot", Amount: int64(ugnotSent)}})
```
Comment (r3v4s) [inline staker/mint_stake.gno:60]: I'm really not sure if its necessary

**Review by r3v4s — APPROVED** [review]
left minor suggestions

File: staker/mint_stake.gno (hunk 3 above)
Comment (notJoon) [reply staker/mint_stake.gno:60]: Not required, but included for readability.

**Review by r3v4s — APPROVED** [review]

**Review by AkydA — APPROVED** [review]

**Pull request merged**